packages/twenty-server/src/engine/core-modules/workflow/filters/workflow-trigger-graphql-api-exception.filter.ts

```diff
@@ -7,18 +9,19 @@ import {
   WorkflowTriggerExceptionCode,
 } from 'src/modules/workflow/workflow-trigger/workflow-trigger.exception';
 
-export const workflowTriggerGraphqlApiExceptionHandler = (error: Error) => {
-  if (error instanceof WorkflowTriggerException) {
-    switch (error.code) {
+@Catch(WorkflowTriggerException)
+export class WorkflowTriggerGraphqlApiExceptionFilter
+  implements ExceptionFilter
+{
+  catch(exception: WorkflowTriggerException) {
+    switch (exception.code) {
       case WorkflowTriggerExceptionCode.INVALID_INPUT:
-        throw new UserInputError(error.message);
+        throw new UserInputError(exception.message);
       case WorkflowTriggerExceptionCode.INVALID_WORKFLOW_TRIGGER:
       case WorkflowTriggerExceptionCode.INVALID_WORKFLOW_VERSION:
       case WorkflowTriggerExceptionCode.INVALID_ACTION_TYPE:
       default:
-        throw new InternalServerError(error.message);
+        throw new InternalServerError(exception.message);
```

Comment:
that's very nice. I think we should do that everywhere else in the code to avoid having two different ways to manage exceptions. Maybe you pass the changes in that PR if it is not too big, or at least create a ticket to do that?